.github/workflows/builds.yml

```diff
@@ -0,0 +1,54 @@
+name: builds
+
+on:
+  release:
+    types: [created]
+  push:
+    branches:
+      - master
+      - main
+  pull_request:
+
+jobs:
+  docker-oss:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            name=rudderlabs/develop-rudder-server
+            name=rudderlabs/rudder-server,enable=${{ github.ref == format('refs/heads/{0}', 'master') }}
```

Comment:
We are consciously omitting OSS image builds for patch releases, correct?

Reply:
That was not intentional actually, I need to enable non-develop images for `release/*` branches as well.